dev-tools/packer/readme.md.j2

```diff
@@ -0,0 +1,23 @@
+# Welcome to {{.beat_name}} {{.version}}
+
+{{.beat_description}}
+
+## Getting Started
+
+To get running with your beat, setup elasticsearch on your localhost and start it with:
+
+     ./{{.beat_name}}  -c {{.beat_name}}.yml -e
+
+This will start the beat and send the data to your elasticsearch instance. To load the dashboards for {{.beat_name}} into Kibana, run:
+
+    ./scripts/import_dashboards
+
+For further steps visit the [Getting started](https://www.elastic.co/guide/en/beats/{{.beat_name}}/current/{{.beat_name}}-getting-started.html) guide.
```

Comment:
Here it's tricky, because "current" is not always correct. For example, if you are on version 5.0.0-beta1, the link will point to the 1.3 documentation.